.github/workflows/pull-request.yml

```
@@ -38,8 +38,12 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
-      - name: Run prettier
-        run: pnpm run lint
+      - name: Setup Biome
+        uses: biomejs/setup-biome@v2
+        with:
+          version: latest
+      - name: Run Biome
```

```suggestion

      - name: Run Biome
```
Nit, linebreak between blocks